CalibTracker/SiStripLorentzAngle/python/EnsembleCalibrationLA_cfg.py

```diff
@@ -7,6 +7,8 @@
 
 from CalibTracker.SiStripLorentzAngle.MeasureLA_cff import METHOD_WIDTH, METHOD_PROB1, METHOD_AVGV2, METHOD_AVGV3, METHOD_RMSV2, METHOD_RMSV3
 
+process.load('Configuration.Geometry.GeometryIdeal_cff')
```

Comment:
Hi @davidlange6, how can I know which one is correct? I have no clue. Maybe @mmusich or @dimattia know?